### PR TITLE
CLOUDP-73729: mongocli ops-manager admin backup s3 describe

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-73729

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

```
./bin/mongocli om admin backup s3 describe andrea.angiolillo.bucket -P ops
ID                         URI                         SSL     LOAD FACTOR   AUTH METHOD
andrea.angiolillo.bucket   mongodb://127.0.0.1:27017   false   50            KEYS
```

```
./bin/mongocli om admin backup s3 describe andrea.angiolillo.bucket -P ops -o json
{
  "id": "andrea.angiolillo.bucket",
  "uri": "mongodb://127.0.0.1:27017",
  "writeConcern": "ACKNOWLEDGED",
  "ssl": false,
  "assignmentEnabled": true,
  "encryptedCredentials": false,
  "loadFactor": 50,
  "awsAccessKey": "XXXXXX",
  "awsSecretKey": "XXXXXXXXXXX",
  "s3AuthMethod": "KEYS",
  "s3BucketEndpoint": "s3-eu-west-1.amazonaws.com",
  "s3BucketName": "andrea.angiolillo.bucket",
  "s3MaxConnections": 50,
  "disableProxyS3": true,
  "acceptedTos": true,
  "sseEnabled": false,
  "pathStyleAccessEnabled": false
}
```

### CODE REVIEW

File: internal/cli/opsmanager/admin/backup/s3/describe.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+var describeTemplate = `ID	URI	SSL	LOAD FACTOR	AUTH METHOD
```
Comment (gssbzn) [inline internal/cli/opsmanager/admin/backup/s3/describe.go:26]: base on the other PR should we change ID for NAME?

**Review by gssbzn — APPROVED** [review]
LGTM one small suggestion but we can do that later

File: internal/cli/opsmanager/admin/backup/s3/describe.go (hunk 1 above)
Comment (andreaangiolillo) [reply internal/cli/opsmanager/admin/backup/s3/describe.go:26]: Once I merged https://github.com/mongodb/mongocli/pull/489 I will fix this here as well

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**